tools/pd-ctl/pdctl/command/gc_safepoint_command.go

```diff
@@ -15,24 +15,18 @@
 package command
 
 import (
-	"encoding/json"
-	"net/http"
 	"sort"
 
 	"github.com/spf13/cobra"
-	"github.com/tikv/pd/server/api"
-)
-
-var (
-	serviceGCSafepointPrefix = "pd/api/v1/gc/safepoint"
 )
 
 // NewServiceGCSafepointCommand return a service gc safepoint subcommand of rootCmd
 func NewServiceGCSafepointCommand() *cobra.Command {
 	l := &cobra.Command{
-		Use:   "service-gc-safepoint",
-		Short: "show all service gc safepoint",
-		Run:   showSSPs,
+		Use:               "service-gc-safepoint",
+		Short:             "show all service gc safepoint",
+		PersistentPreRunE: requirePDClient,
```

Comment:
Does `NewDeleteServiceGCSafepointCommand` need to add `requirePDClient`?

Reply:
It's a subcommand of NewServiceGCSafepointCommand, so it inherits PersistentPreRunE: requirePDClient

Reply:
got it.